ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -168,8 +168,85 @@ SettingsContentBase {
         }
 
         Rectangle {
-            id: modeSeparator
+            id: scrollSeparator
             anchors.top: zoomSlider.bottom
+            anchors.topMargin: Style.current.padding * 3
+            anchors.left: parent.left
+            anchors.right: parent.right
+            height: 1
+            color: Style.current.separator
+        }
+
+        StatusSectionHeadline {
+            id: labelScrolling
+            anchors.top: scrollSeparator.bottom
```

Comment:
I understand that the usage of `anchors.top: x.bottom` adheres to the conventions established in this file, but please consider a refactor utilizing layouts as more maintainable and readable approach.